src/Menu/Menu.js

```diff
@@ -63,7 +63,8 @@ export const MenuItem = ({ url, link, isLink, separator, addon, children, onclic
                 }
                 {(!url && !link) && <a {...linkProps}
                     className='fd-menu__item'
-                    onClick={onclick}>{children}</a>}
+                    onClick={onclick}
+                    role='button'>{children}</a>}
```

Comment:
Although `<a>` elements with a role of `button` "works", is it the best choice?  @jeffredodd What have we typically done with menu items in the past?  Are they anchors or buttons?

Reply:
actually, i think they are supposed to have the role of `menuitem`

I think for things like this, fundamentals should show in their example, the proper roles.